hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/TestGrpcOzoneManagerServer.java

```diff
@@ -55,9 +57,11 @@ public void testStartStop() throws Exception {
     try {
       server.start();
     } catch (Exception e) {
+      ex = new Exception(e);
       e.printStackTrace();
     } finally {
       server.stop();
+      Assert.assertNull(ex);
```

Comment:
Nit: I think it's a bit too complicated.  Simply not catching the exception would cause the test to fail and let the test runner log the exception.  Just drop the `catch` block.